tremor-script/src/grammar.lalrpop

```diff
@@ -23,7 +24,172 @@ use beef::Cow;
 
 grammar<'input>;
 
-////////////////////////////// Root statements //////////////////////////////
+////////////////////////////// Troy statements //////////////////////////////
+// These statements for the root of the language.
+
+/// Root
+pub(crate) Deploy : DeployRaw<'input> = {
+    <config:ConfigDirectives> <stmts:DeployStmts> "<end-of-stream>"? => DeployRaw{config, stmts},
```

Comment:
We probably want a doc comment variant too so deployments can be documented?

Reply:
Now `how` did I miss this? Yes, we absolutely do!

Reply:
I think because we entirely missed it with trickle as well :D until Yatin pointed it out I was absolutely oblivious to that too

Reply:
Added